packages/loader/container-loader/src/container.ts

```diff
@@ -560,7 +556,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 all: {
                     clientType, // Differentiating summarizer container from main container
                     containerId: uuid(),
-                    docId: () => this.id,
+                    docId: () => this._resolvedUrl?.id ?? "",
```

Comment:
I'd prefer just this._resolvedUrl?.id here, or (if it can be null) to have this._resolvedUrl?.id ?? undefined.
There is no benefit of recording empty string in telemetry, undefined (which is dropped) is pretty good default value.